Examples/test-suite/li_math.i

```diff
@@ -4,4 +4,8 @@
 // warning.
 %warnfilter(SWIGWARN_PARSE_KEYWORD);
 #endif
+#ifdef SWIGFORTRAN
+%warnfilter(SWIGWARN_PARSE_BUILTIN_NAME);
```

Comment:
This warning suppression doesn't seem to be needed.

Reply:
Yeah, I didn't realize when I made this test that `WARN_PARSE_BUILTIN_NAME` (321) is explicitly disabled by default with SWIG. I'll remove.